flake.nix

```diff
@@ -0,0 +1,43 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
+    nixgl.url = "github:nix-community/nixGL";
+    nixgl.inputs.nixpkgs.follows = "nixpkgs";
+  };
+  outputs = {self, nixpkgs, nixgl}:
+    let
+      system = "x86_64-linux";
+      pkgs = import nixpkgs {
+        inherit system;
+        overlays = [ nixgl.overlay ];
+        config = { allowUnfree = true; };
+      };
+    in
+    {
+      packages.${system} = {
+        tangerine = pkgs.stdenv.mkDerivation {
+          name = "tangerine";
+          patches = [
+            (pkgs.fetchpatch {
+              url = "https://github.com/Aeva/tangerine/pull/12/commits/2d7d1ae1e21e8fe52df2c4a33e947b2ff6b07812.patch";
```

Comment:
this would be your other pull request, correct?

Reply:
Correct - once it's merged / updated, the list here can be updated.